### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mesrikanthreddy**
Updated with grammatical updates: This is my first PR, and I am looking forward to contributing more.

Fixes #

## Target Release

1.5.x

### NEW FEATURES | UPGRADE NOTES | ENHANCEMENTS | BUG FIXES | EXPERIMENTS

 

-

### CODE REVIEW

**Comment by crw** [comment]
Thanks for the submission! I've put it on the docs team triage queue. Since these changes deal with style, the docs team has the final say on these types of changes. Thanks again!

**Comment by Jagannathmaity110** [comment]
Is this  issue solved ?

File: README.md [hunk 1]
```diff
@@ -8,19 +8,19 @@
...
 
 The key features of Terraform are:
 
-- **Infrastructure as Code**: Infrastructure is described using a high-level configuration syntax. This allows a blueprint of your datacenter to be versioned and treated as you would any other code. Additionally, infrastructure can be shared and re-used.
+- **Infrastructure as Code**: Infrastructure is described using a high-level configuration syntax. This allows a blueprint of your data center to be versioned and treated as you would any other code. Additionally, infrastructure can be shared and re-used.
 
-- **Execution Plans**: Terraform has a "planning" step where it generates an execution plan. The execution plan shows what Terraform will do when you call apply. This lets you avoid any surprises when Terraform manipulates infrastructure.
+- **Execution Plans**: Terraform has a "planning" step where it generates an execution plan. The execution plan shows what Terraform will do when you call to apply. This lets you avoid any surprises when Terraform manipulates infrastructure.
```
Comment (trujillo-adam) [inline README.md:17]:
```suggestion
- **Execution Plans**: You can generate an execution plan that previews the infrastructure that Terraform builds when you apply the configuration. This lets you avoid any surprises when Terraform manipulates infrastructure.
```
The existing description wasn't consistent with our style guidelines

File: README.md [hunk 2]
```diff
@@ -8,19 +8,19 @@
...
 
-- **Resource Graph**: Terraform builds a graph of all your resources, and parallelizes the creation and modification of any non-dependent resources. Because of this, Terraform builds infrastructure as efficiently as possible, and operators get insight into dependencies in their infrastructure.
+- **Resource Graph**: Terraform builds a graph of all your resources and parallelizes creating and modifying any non-dependent resources. Because of this, Terraform builds infrastructure as efficiently as possible, and operators get insight into dependencies in their infrastructure.
```
Comment (trujillo-adam) [inline README.md:19]:
```suggestion
- **Resource Graph**: Terraform builds a graph of all your resources and parallelizes the creation and modification of non-dependent resources. As a result, Terraform builds infrastructure as efficiently as possible and helps operators get insight into infrastructure dependencies.
```
The grammar in this one is kind of weird. Without getting into the weeds, I think your edit is technically correct, but I think we should revert it for clarity.

File: README.md [hunk 3]
```diff
@@ -8,19 +8,19 @@
...
 
 - **Change Automation**: Complex changesets can be applied to your infrastructure with minimal human interaction. With the previously mentioned execution plan and resource graph, you know exactly what Terraform will change and in what order, avoiding many possible human errors.
 
-For more information, refer to the [What is Terraform?](https://www.terraform.io/intro) page on the Terraform website.
+If you want more information, please refer to the [What is Terraform?](https://www.terraform.io/intro) page on the Terraform website.
```
Comment (trujillo-adam) [inline README.md:23]:
```suggestion
For additional information, refer to [What is Terraform?](https://developer.hashicorp.com/terraform/intro) on the Terraform website.
```
Per our style guide, avoid using unnecessary words. This is a good opportunity to update the link, though.

File: README.md [hunk 4]
```diff
@@ -29,19 +29,19 @@ Documentation is available on the [Terraform website](https://www.terraform.io):
 - [Introduction](https://www.terraform.io/intro)
 - [Documentation](https://www.terraform.io/docs)
 
-If you're new to Terraform and want to get started creating infrastructure, please check out our [Getting Started guides](https://learn.hashicorp.com/terraform#getting-started) on HashiCorp's learning platform. There are also [additional guides](https://learn.hashicorp.com/terraform#operations-and-development) to continue your learning.
+If you're new to Terraform and want to create infrastructure, please check out our [Getting Started guides](https://learn.hashicorp.com/terraform#getting-started) on HashiCorp's learning platform. To continue your learning, there are also [additional guides](https://learn.hashicorp.com/terraform#operations-and-development).
```
Comment (trujillo-adam) [inline README.md:32]:
```suggestion
If you are new to Terraform, refer to the [getting started guides](https://developer.hashicorp.com/terraform/tutorials#get-started) to start creating infrastructure. To continue learning about Terraform, browse the [tutorial library](https://developer.hashicorp.com/terraform/tutorials).
```

File: README.md [hunk 5]
```diff
@@ -29,19 +29,19 @@ Documentation is available on the [Terraform website](https://www.terraform.io):
...
 
 ## Developing Terraform
 
-This repository contains only Terraform core, which includes the command line interface and the main graph engine. Providers are implemented as plugins, and Terraform can automatically download providers that are published on [the Terraform Registry](https://registry.terraform.io). HashiCorp develops some providers, and others are developed by other organizations. For more information, see [Extending Terraform](https://www.terraform.io/docs/extend/index.html).
+This repository contains only the Terraform core, which includes the command line interface and the main graph engine. Providers are implemented as plugins, and Terraform can automatically download published providers on [the Terraform Registry](https://registry.terraform.io). HashiCorp develops some providers, and other organizations develop others. For more information, see [Extending Terraform](https://www.terraform.io/docs/extend/index.html).
```
Comment (trujillo-adam) [inline README.md:38]:
```suggestion
This repository contains only Terraform core, which includes the command line interface and the main graph engine. Providers are implemented as plugins, and Terraform can automatically download published providers from [the Terraform Registry](https://registry.terraform.io). The registry contains third-party and HashiCorp-developed providers. For more information, refer to [Plugin Development](https://developer.hashicorp.com/terraform/plugin) in the Terraform documentation.
```

File: README.md [hunk 6]
```diff
@@ -29,19 +29,19 @@ Documentation is available on the [Terraform website](https://www.terraform.io):
...
 
-- To learn more about compiling Terraform and contributing suggested changes, refer to [the contributing guide](.github/CONTRIBUTING.md).
+- Refer to [the contributing guide](.github/CONTRIBUTING.md) to learn more about compiling Terraform and contributing suggested changes.
 
-- To learn more about how we handle bug reports, refer to the [bug triage guide](./BUGPROCESS.md).
+- To learn more about how we handle bug reports, please look at the [bug triage guide](./BUGPROCESS.md).
```
Comment (trujillo-adam) [inline README.md:42]:
```suggestion
- Refer to the [bug triage guide](./BUGPROCESS.md) to learn more about how we handle bug reports.
```
We should phrase these to be consistent with your suggestion at line 40.

File: README.md [hunk 7]
```diff
@@ -29,19 +29,19 @@ Documentation is available on the [Terraform website](https://www.terraform.io):
...
 
-- To learn how to contribute to the Terraform documentation in this repository, refer to the [Terraform Documentation README](/website/README.md).
+- To learn how to contribute to the Terraform documentation in this repository, please look at the [Terraform Documentation README](/website/README.md).
```
Comment (trujillo-adam) [inline README.md:44]:
```suggestion
- Refer to the [Terraform documentation README](/website/README.md) to learn how to contribute to the Terraform documentation in this repository.
```

**Review by trujillo-adam — COMMENTED** [review]
Thanks for helping out with this documentation. Some of your edits are inconsistent with our writing styles, but you have given us an opportunity to make improvements by opening this PR.

**Comment by Yashrajput7232** [comment]
i want to resolve this issue.

**Comment by crw** [comment]
Closing due to stale PR. This could still be submitted, so if you decide you want to follow-through @mesrikanthreddy, please let me know and I can re-open. Thanks!

**Pull request closed without merging**